src/components/cdPipeline/NewCDPipeline.tsx

```diff
@@ -804,6 +804,7 @@ export default function NewCDPipeline({
                         const form = { ...formData }
                         form.clusterName = response.result.deleteResponse?.clusterName
                         setFormData(form)
+                        openDeleteModal()
```

Comment:
Is it intentional?

Reply:
yes, this is for cluster unreachable case